packages/backend/src/util/callWithRetry.ts

```diff
@@ -0,0 +1,22 @@
+function wait(ms: number) {
+  return new Promise(res => setTimeout(res, ms));
+}
+
+const MAX_NUMBER_OF_RETRIES = 5;
+
+export async function callWithRetry<T>(
```

Comment:
just noting that we have an exponential backoff helper in clerk-js so we might want to combine both and move into shared in the future

Reply:
Added a TODO